cider-velox/src/CiderOperator.cpp

```diff
@@ -90,9 +96,23 @@ void CiderOperator::addInput(RowVectorPtr input) {
   }
 
   input_ = std::move(input);
-  auto inBatch =
-      dataConvertor_->convertToCider(input_, input_->size(), &convertorInternalCounter);
-  ciderRuntimeModule_->processNextBatch(inBatch);
+  if (is_using_arrow_fromat_) {
+    for (size_t i = 0; i < input_->childrenSize(); i++) {
+      input_->childAt(i)->mutableRawNulls();
```

Comment:
What is this loop for?

Reply:
It's so weird

Reply:
workaround, process cannot support no null buffer